modules/service/deployment/0.1/facets.yaml

```
@@ -353,19 +355,121 @@ spec:
+              description: Commands for startup check
+              items:
+                type: string
+              x-ui-override-disable: true
           required:
-            - readiness_port
-            - readiness_url
-            - readiness_exec_command
-            - liveness_port
-            - liveness_url
-            - liveness_exec_command
+          - readiness_port
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)